.github/workflows/pulumi.deploy.yml

```
@@ -0,0 +1,61 @@
+          node-version: 20
+
+      - name: Install NPM dependencies
+        run: npm install
+
+      - name: Install Pulumi CLI
+        uses: pulumi/setup-pulumi@v2
+        with:
+          pulumi-version: ^3.0.0
+
+      - name: Authenticate to GCP
+        uses: google-github-actions/auth@v1
```

actionlint (1.7.4)

the runner of "google-github-actions/auth@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)